doc-content/shared-kie-docs/src/main/asciidoc/Workbench/SecurityManagement/business-central-settings-changing-permissions-proc.adoc

```diff
@@ -0,0 +1,31 @@
+[id='business-central-settings-changing-permissions-proc_{context}']
+= Changing permissions for groups and roles in {CENTRAL}
+
+In {CENTRAL}, you cannot change permissions for an individual user. However, you can change permissions for groups and roles. The changed permissions apply to users that have the role or belong to the group that you changed.
+
+[NOTE]
+====
+Any changes you make to roles or groups affect all of the users associated with that role or group.
+====
+
+.Procedure
+. To access the *Security management* page in {CENTRAL}, select the *Admin* icon in the top-right corner of the screen.
+. Click *Roles*, *Groups*, or *Users* on the {CENTRAL} *Settings* page.
++
+The *Security management* page opens on the tab for the icon that you clicked.
+. From the list, click the role or group you want to edit. All details are displayed in the right panel.
+. Set the *Home Page* or *Priority* under the *Settings* section.
+. Set the {CENTRAL}, page, editor, space, and project permissions under the *Permissions* section.
++
+.Setting the permissions
+image::Workbench/SecurityManagement/set-permissions.png[Setting the permissions]
++
+You can expand the resource type whose permissions you want to change.
```

Comment:
You can expand the resource type whose permissions you want to change.
Change to:
6. Click the arrow next to a resource type to (and I'm not sure just what happens when you do this or why you would want to...)

Reply:
When you click the arrow next to a resource type it expands the resource type and you can change the permission by using the slider button.